### ISSUE TITLE
Make `solve` returns more reproducible

### ISSUE THREAD

**Issue by Davide-sd** [gutow/Algebra_with_Sympy#27]
Consider this example:

```py
from sympy import *
from algebra_with_sympy import Eqn, solve, algwsym_config
algwsym_config.output.label = False
algwsym_config.output.solve_to_list = True
init_printing()

xi, wn = symbols("xi omega_n", real=True, positive=True)
Tp, Ts = symbols("T_p, T_s", real=True, positive=True)
e1 = Eqn(Tp, pi / (wn*sqrt(1 - xi**2)))
e2 = Eqn(Ts, 4 / (wn*xi))
display(e1, e2)
```
![image](https://github.com/gutow/Algebra_with_Sympy/assets/9921777/67329ffd-58c8-4ffc-bf6e-dacf1ec44d6c)

I would like to solve the two equations for `xi, wn`. Note that I'm returning lists of solutions:

```py
sol = solve([e1, e2], [xi, wn])[0]
xi_eq, wn_eq = sol
display(sol, xi_eq, wn_eq)
```
![image](https://github.com/gutow/Algebra_with_Sympy/assets/9921777/9712d67f-1c04-44e5-ab40-746fcc6297f2)

Note that I specified `[xi, wn]` in the `solve` command, however the solutions returned by `solve` are swapped.

This happens because the `solve` command internally calls sympy's `solve` function requesting a dictionary output:

```py
    flags['dict'] = True
    result = solve(newf, *symbols, **flags)
```

All the solutions are added to the `solns` list. It would be nice to sort them according to the user-provided symbols. This would make for a terrific improvements, because then this `solve` function could be used inside custom functions and always get reproducible results, which is not the case with the current implementation.

**Comment by Davide-sd** [comment]
I'm going to attempt a fix in a couple of days...

**Comment by gutow** [comment]
@Davide-sd As you have said you will work on this, I have assigned it to you. The override of solve is still in this package. The definition of the `Equation` type and some of the output rendering code is now in the custom version of sympy. This was necessary to avoid issues with sympy using internal versions of operations and functions that could not be overridden by an external package. Thanks for working on this.

**Comment by gutow** [comment]
This will be in release v1.0.1

**Comment by gutow** [comment]
fixed in v1.0.1
